gradle/test.gradle

```diff
@@ -37,6 +37,7 @@ task runTestsWithTag(type: Test) {
             // used to get the default docker image id.
             systemProperty("TERRA_JAR_IMPLEMENTATION_VERSION",
                     "${project.properties['dockerRepoPath']}/${project.properties['dockerImageName']}/${project.version}:${project.properties['dockerImageTag']}")
+            systemProperty("TERRA_CLI_VERSION", "${project.version}")
```

Comment:
// Normally CLI version comes from jar. For unit tests, there is no jar, so pass version this way.